MIGRATION-FROM-SCK.md

```diff
@@ -0,0 +1,349 @@
+# Migration from Splunk Connect for Kubernetes
+
+## Assumptions
+
+You are running Splunk Connect for Kubernetes (SCK) 1.4.9 and want to migrate to Splunk OpenTelemetry Collector for Kubernetes. Previous versions of the SCK are out of scope, but this guide still applies. You might encounter some version specific issues (such as missing/changed config options) but that shouldn't prevent you from proceeding with migration.
+
+## Changes in components
+
+SCK has 3 components/applications:
+
+1. Logs, metrics and traces from a Kubernetes cluster (deployed as a DaemonSet)
+2. Application to fetch cluster metrics from a kubernetes cluster (deployed as a deployment)
+3. Application to fetch kubernetes objects metadata from a kubernetes cluster (deployed as a deployment)
+
+All SCK applications use Fluentd to work with logs, metrics and objects. Fluentd has significant performance issues when used to fetch logs in a Kubernetes cluster with pods that have very high throughput.
+
+Splunk OpenTelemetry Collector for Kubernetes provides significant performance improvements over the SCK through use of the OpenTelemetry Collector agent and native OpenTelemetry functionality for logs collection rather than Fluentd. You can learn more about the performance characteristics of this new application [Performance of native OpenTelemetry logs collection](https://github.com/signalfx/splunk-otel-collector-chart/blob/main/docs/advanced-configuration.md#performance-of-native-opentelemetry-logs-collection).
+
+Splunk OpenTelemetry Collector for Kubernetes has the following components and applications:
+
+1. Application to fetch logs and traces from a kubernetes cluster (deployed as a DaemonSet)
+2. Application to fetch metrics and objects from a kubernetes cluster (deployed as a DaemonSet).
+
+There is no application available for fetching Kubernetes objects metadata from a Kubernetes cluster.
+
+### Changes in logs, metrics, and objects
+
+| Logging | Metrics | Objects |
+|---|---|---|
+| Redhat UBI docker images for our applications are no longer available, as we now use scratch images | The naming convention of the metrics used in Splunk OpenTelemetry Collector for Kubernetes has changed and there are minor differences in the names of the metrics | Not implemented or available |
+| AWS FireLens is not supported. | Changes in the number of metrics (fewer metrics available) and additional metrics for the OpenTelemetry Collector |  |
+|  |  |  |
```

Comment:
This section seems redundant as everything is repeated below. Up to you to keep it or not

Reply:
it is redundant. i am okay either way. I will leave it until someone from doc comments

Reply:
Agree with Dmitrii. I would remove the table.